network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -90,17 +90,13 @@ <h2 class="h4-heading">{% trans "Trending projects" %}</h2>
       <a class="cta-link d-block d-md-none" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
     </div>
   </div>
-  <div class="row section-quote">
+  <div class="row section-quote d-flex align-items-center">
```

Comment:
@KalobTaulien my previous PR has passed design review. If we are making any design adjustment, let's find designers for a quick review 👀

Reply:
@mmmavis Good point. This just centers the content if the image is too big and theres not enough text. 

@taisdesouzalessa thoughts on this?

Reply:
@KalobTaulien I am not sure which component you are all talking about it here...do you have a screenshot? Tks!

Reply:
@taisdesouzalessa sorry for the late reply here. 

![image](https://user-images.githubusercontent.com/4743971/88821988-88d68280-d180-11ea-8a74-8e44d354bd5d.png)
It's this section. If the content gets too long the image sits at the top instead of centered with the content. I'm proposing we add vertical centering. We might not even see this as an issue if the content stays short.

Reply:
Making a note that we spoke about this already. I'm resolving this. Thanks @taisdesouzalessa!